### DISCUSSION TITLE
problem with vercel's deployment

### DISCUSSION THREAD

**Discussion by MitkoVtori** [anuraghazra/github-readme-stats#2329] [Q&A] [answered]
Hey, I am not sure what is happening, but from that I understand from google, when I change the vercel.json in a fork and make a pull request (see #2324) a **Team member** should approve or something. 
Is there a way I can fix this my self, or this is just like the normal github approval but in vercel?

**Comment by rickstaa** [comment, marked as answer]
@MitkoVtori Thanks for your pull request. Because of security concerns, Vercel requires team members to approve deployments on PRs. However, since in #2324, you only changed markdown files, you don't need to test the deployment. Since we are changing to a new (automatic) translation system, I did not merge any translation-related PRs in the last months (see https://github.com/anuraghazra/github-readme-stats/issues/2053).
